src/components/video/SimpleVideo.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
```

Comment:
We rather don't use this type of imports
```suggestion
import React from 'react';
```